ext/drmgr/drmgr.h

```diff
@@ -1272,8 +1272,10 @@ DR_EXPORT
  *
  * See also drmgr_unregister_signal_event().
  */
+/* clang-format off */ /* (work around clang-format newline-after-type bug) */
 bool drmgr_unregister_signal_event_user_data(
```

Comment:
You need to fix the style too: put the newline after `bool`

Reply:
Okay no problem. Shall I do the same for the following?

```
bool drmgr_unregister_signal_event(dr_signal_action_t (*func)(void *drcontext,
                                                              dr_siginfo_t *siginfo));
```

Reply:
Done. Let me know if this needs to be changed back.